webapp/src/gitjson.tsx

```diff
@@ -693,6 +684,17 @@ class NoChangesComponent extends sui.StatelessUIElement<GitHubViewProps> {
                     </p>
                     <sui.Button className="primary" text={lf("Create release")} onClick={this.handleBumpClick} onKeyDown={sui.fireClickOnEnter} />
                 </div> : undefined}
+            {master && needsLicenseMessage ? <div className="ui message">
+                <div className="content">
+                    {lf("Your project doesn't seem to have a license. This makes it hard for others to use it.")}
+                    {" "}
```

Comment:
could this be combined with previous line?

Reply:
no it messes up translations.